draft-ietf-quic-qpack.md

```diff
@@ -273,20 +285,20 @@ permitted by the value of SETTINGS_QPACK_BLOCKED_STREAMS, compression efficiency
 can often be improved by referencing dynamic table entries that are still in
 transit, but if there is loss or reordering the stream can become blocked at the
 decoder.  An encoder can avoid the risk of blocking by only referencing dynamic
-table entries which have been acknowledged, but this could mean using
-literals. Since literals make the header block larger, this can result in the
-encoder becoming blocked on congestion or flow control limits.
+table entries which have been acknowledged, but this could mean using literals.
+Since literals make the encoding of the field section larger, this can result in
```

Comment:
"Encoding of the field section" makes one think of the process rather than the binary result.  "Encoded field section" might be more appropriate here.